main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: the cat sprints
+cat.sprint();
+//  action 2: the cat scratches
+cat.scratch();
+//  action 3: the cat eats
+cat.eat();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: printer
+//  action 1: the printer prints documents
+printer.printDocument();
+//  action 2: the printer scan documents
+printer.scanDocument();
+//  action 3: the printer sends a fax
+printer.fax();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: player (video games)
+//  action 1: the player runs
+player.run();
+//  action 2: the player attacks
+player.attack();
+//  action 3: the player heals himself/herself
+player.heal();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: phone
+//  action 1: the phone takes a photo
+phone.takePhoto();
+//  action 2: the phone plays audio
+phone.playAudio();
+//  action 3: the phone sends out a message
+phone.sendMessage();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bus 
+//  action 1: the bus takes passengers
+bus.takePassenger();
+//  action 2: the bus consumes fuel
+bus.consumeFuel();
+//  action 3: the bus charges the passengers for ticket
+bus.chargeTicket();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bat
+//  action 1: the bat flies
+bat.fly();
+//  action 2: the bat hunts
+bat.hunt();
+//  action 3: the bat sleeps
+bat.sleep();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.